docs/user_guide.rst

```diff
@@ -7,20 +7,70 @@ User Guide
 .. contents:: Table of contents:
    :local:
 
-IO
-==
+
+.. _reading_genetic_data:
+
+Reading genetic data
+====================
+
+Installation
+------------
+
+Sgkit can read standard genetic file formats, including VCF, PLINK, and BGEN. Support for reading
+these formats is not installed by default, and requires additional dependencies, which can be installed
+as an "extra" feature using pip, as follows.
+
+To install sgkit with BGEN support::
+
+    $ pip install 'sgkit[bgen]'
+
+To install sgkit with PLINK support::
+
+    $ pip install 'sgkit[plink]'
+
+To install sgkit with VCF support::
+
+    $ pip install 'sgkit[vcf]'
+
```

Comment:
Question: do we have an 'sgkit[all]' that'll pull everything in? That'd be handy, and good to document.

Reply:
We don't have an `sgkit[all]`.